soapui/src/main/java/com/eviware/soapui/actions/SumbitUserInfoAction.java

```diff
@@ -48,9 +52,15 @@ public class SumbitUserInfoAction {
     private static final String EMAIL_HINT = "Enter e-mail";
     private static final String DIALOG_CAPTION = "Stay Tuned!";
     private static final String DIALOG_MAIN_TEXT = "Want to stay in the loop?";
-    private static final String DIALOG_DESCRIPTION = "We will use this information to help you get started, and provide you with best practices we’ve learned from over 7.5 Million development, quality and operations experts just like you.";
+    private static final String DIALOG_DESCRIPTION = "Your privacy matters. This information will be used to provide you " +
```

Comment:
Do we concatenate this purely for readability reasons?

Reply:
Yes